src/main/Config.h

```diff
@@ -121,9 +121,9 @@ class Config : public std::enable_shared_from_this<Config>
     // application config
 
     // The default way stellar-core starts is to load the state from disk and
-    // catch up to the network before starting SCP. If you need different
-    // behavior you need to use new-db or force-scp which sets the following
-    // flags:
+    // start a consensus round (if node is validating), then maybe trigger
+    // catchup. If you need different behavior you need to use new-db or
+    // --wait-for-consensus option which sets the following flag:
```

Comment:
comment was not properly updated. "wait-for-consensus" _clears_ the flag ; also here is a "all caps" that this should not be set in a config file - we should remove that as well

Reply:
updated (squashed as well)